lib/chibi/repl.sld

```diff
@@ -8,6 +8,7 @@
           (srfi 1)
           (srfi 9)
           (only (srfi 18) current-thread)
+          (srfi 35 internal)
```

Comment:
My understanding was that R7RS Large was actually going in a different direction from R6RS conditions?

R7RS small uses opaque predicates, and John wanted a list of condition-types: https://github.com/johnwcowan/r7rs-work/blob/master/ConditionsCowan.md

I don't want to pull anything into the core which is at odds with R7RS.  I also don't want to bloat the core.  Many libraries are going to want to provide nicer pretty printing for the REPL (I want this for arrays).  To manage this we should either use a generic which can be overridden per type, or provide hooks which would be called by the other libraries (i.e. on loading SRFI 35 it would register itself into the REPL).